presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1638,4 +1640,17 @@ public Duration getPartitionLeaseDuration()
     {
         return partitionLeaseDuration;
     }
+
+    @Config("hive.missingPartitionPredicateThreshold")
+    @ConfigDescription("Max number of partition predicates that can be missing before the query string becomes too long")
```

Comment:
We need to have materialized view in the description and config for the context.